### ISSUE TITLE
Gutenberg toggle should only have effect on new posts.

### ISSUE THREAD

**Issue by koke** [wordpress-mobile/gutenberg-mobile#565]
Existing posts should open in Gutenberg if they have block content, or Aztec otherwise.

- [x] iOS side work
- [x] Android side work

**Comment by hypest** [comment]
OK, let's clear up the design on this one.

We're looking at combining the two distinct settings currently in the App Settings (the editor selection and the Gutenberg switch) into only one and to only govern which editor to use for _new posts_, right?

We can have the new UI be a single switch/toggle (Gutenberg when ON, Aztec when OFF) or a selection. Any preference? Let's go for the simple switch?

cc @iamthomasbishop

**Comment by koke** [comment]
There’s some existing discussion on #567 about the copy, but yes. Only the switch should remain with Gutenberg as the ON value

**Comment by hypest** [comment]
I'm planning to position/move the toggle into the place the editor selection was before:

![screenshot-1549879746323](https://user-images.githubusercontent.com/1032913/52558908-789bdd80-2dfc-11e9-8d3c-7e961aa4f049.jpg)

**Comment by koke** [comment]
👍 makes sense. Some day we should order the sections consistently across platforms, but not today 😉

**Comment by iamthomasbishop** [comment]
Just want to clarify and give a visual example here. Is this in-line with what we are considering?

<img width="929" alt="image" src="https://user-images.githubusercontent.com/1200432/52573192-3912c880-2ddf-11e9-966e-bff22cc9cbf9.png">

**Comment by hypest** [comment]
Haven't added the second line in the PR I have already open but, happy to add one.

**Comment by koke** [comment]
I thought we were going to use something like "Use Block Editor for New Posts" and use the explanation to showcase that they can still "Switch to Classic" from any post

**Comment by iamthomasbishop** [comment]
> I thought we were going to use something like "Use Block Editor for New Posts"

I started there and used "Use Block Editor for New Posts/Pages", but that was too long for the label, and considering it applies to both post types, I removed the "for new..." bit. If we can bump the label to two lines then that's solved.

> use the explanation to showcase that they can still "Switch to Classic" from any post

Same concern re: length, but if we feel it's worth having a 4-line descriptor there then wfm.

**Comment by koke** [comment]
OK, that makes sense. I'd tweak the wording of the explanation a bit, I find the current one a bit hard to parse, and I would skip mentioning existing posts:

"Edit new posts and pages with the Block Editor"

BTW, title case looks fine in the label but odd in the explanation, but I've always found title case to an odd style choice in English 😁 Everything I'm finding so far has "Block Editor" in titles but "block editor" in regular text ([example](https://woocommerce.com/posts/woocommerce-and-wordpress-5-0/#)).

**Comment by iamthomasbishop** [comment]
> "Edit new posts and pages with the block editor"

Sounds good to me

> BTW, title case looks fine in the label but odd in the explanation

Agree, I meant only title case in the cell title – normal case in the explanation. I'd bump the description for copy review, because I'm not 100% sure on the "Block Editor" vs "block editor" – I would assume the former because it's a noun, but I'm by no means an English major 😄
